lib/rbkit/server.rb

```diff
@@ -7,7 +7,7 @@ def initialize(pub_port, request_port)
       @request_port = request_port.to_i
       [@pub_port, @request_port].each{|port| validate_port_range(port) }
       @profiler_thread = nil
-      @profiler_stop_thread = false
+      @stop_profiler_thread = false
```

Comment:
We can't use `get_queued_messages` after stopping the server unfortunately, because stopping the server will clear the queued messages. We can test that `send_hash_as_event` does not cause a segmentation fault, just to prevent any regression. But if it was passing without this change, something is wrong isn't it ?